packages/data/addon/serializers/elide-metadata.ts

```diff
@@ -215,8 +215,9 @@ export default class ElideMetadataSerializer extends EmberObject {
    * Runtime typecheck that typescript can understand
    * @param payload
    * @return true if payload is an instance of TablePayload
-   */
-  isTablePayload(payload: TablePayload): payload is TablePayload {
-    return !!(payload as TablePayload).tables;
+   * */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Maybe want to disable this in the `.eslintrc.js` for the data package
https://github.com/yahoo/navi/blob/132018b4d301abc09990336cd3a63c215fc402a1/packages/data/.eslintrc.js#L23-L26

Reply:
I think it might be nice to keep while we're still converting js files. It prevents us from getting too lazy